src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -267,6 +267,29 @@ export function transformProviderFacilities(providerFacilities) {
   }));
 }
 
+export function concatIncidentLocationString(incidentLocation) {
+  return [
+    incidentLocation.city,
+    incidentLocation.state,
+    incidentLocation.country,
+    incidentLocation.additionalDetails,
+  ]
+    .filter(locationField => locationField && locationField.length > 0)
+    .join(', ');
+}
+
+export function getPtsdChangeFieldTitles(changeFields, formConfig) {
+  const titles = [];
+  Object.keys(changeFields)
+    .filter(
+      key => key !== 'other' && key !== 'otherExplanation' && formConfig[key],
+    )
+    .forEach(key => {
```

Comment:
Is there any way this could be a map instead, and just `return Object.keys(fields).filter(key => .....).map(key => .....)`. Or maybe even a `reduce()` ?